decksite/sql/47.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS rule (
```

Comment:
`rule` feels pretty generic. `archetype_rule`? Or does that imply it's a join table of archetypes and rules?

Reply:
Previously you suggested a single-word name for tables that represent a single entity. Of course, arguably a rule is a join of an archetype and a rule, we just don't bother tracking the 'rule' pay because it contains zero information and the relationship is one-to-many. I don't know if we're ever sufficiently likely to have anything else called `rule` that this has any potential ambiguity to make it worth making all the queries even more verbose.

Reply:
Yeah, neither thing makes me totally happy. Rule is just such a very general word. But I guess there's not much point in changing it to something longer which is also problematic.

Are any of classification/codification/systematization/codification/allocation/designation/regulation/assignment better? Probably not.